packages/@react-spectrum/provider/docs/Provider.mdx

```diff
@@ -0,0 +1,78 @@
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/provider';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Button} from '@react-spectrum/button';
+import {Checkbox} from '@react-spectrum/Checkbox';
+```
+
+# Provider
+
+<p>{docs.exports.Provider.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Provider']}
+  sourceData={[]} />
```

Comment:
*in case you didn't see Ross' Well documentation pull 
sourceData now has a default value of an empty array so it should be ok removing sourceData here if you aren't using it